run_dir/static/css/status_b5.css

```diff
@@ -149,6 +149,7 @@ h1 small, h2 small, h3 small, h4 small{
 .bg-imp  {background-color: #a62100;}
 .bg-devi  {background-color: #a13a7e;}
 .bg-usr  {background-color: #ff6f00;}
+.bg-succe {background-color: #5cb85c;}
```

Comment:
You could have used the existing `.btn-succe`, probably would have the same effect.

Reply:
True, but I realized that the color of the "Lab"-button in the rn's, isn't the same as for the button, so maybe it should be there? It looks nicer when they are the same color and different from the "Flowcell"-button. 

I mean here: https://github.com/SciLifeLab/genomics-status/blob/eb62165d2bf78e2d6c8e9b4deb66f7175ff504ed/run_dir/static/js/running_notes.js#L15

Reply:
Also, `bg-succe` is needed for the dictionary now 😄